package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "@ijmbarr/jupyterlab_spellchecker",
-  "version": "0.1.10",
+  "name": "@ocordes/jupyterlab_spellchecker",
+  "version": "0.1.12",
   "description": "A spell checker for jupyterlab.",
   "keywords": [
     "jupyter",
     "jupyterlab",
     "jupyterlab-extension"
   ],
-  "homepage": "https://github.com/ijmbarr/jupyterlab_spellchecker",
+  "homepage": "https://github.com/ocordes/jupyterlab_spellchecker",
   "bugs": {
-    "url": "https://github.com/ijmbarr/jupyterlab_spellchecker"
+    "url": "https://github.com/ocordes/jupyterlab_spellchecker"
   },
   "license": "BSD-3-Clause",
-  "author": "Iain Barr",
+  "author": "Iain Barr + Oliver Cordes",
```

Comment:
I think that package.json should remain unchanged, except for adding your name into `contributors` property (which needs to be created - an array of strings, or persons as defined in the [npmjs documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)) if you wish to have the recognition.

Reply:
Please, do what is best, I'm new to npmjs and this was a quick&dirty hack to get this running in my production enviroment.